### ISSUE TITLE
Enable scrollable-region-focusable	Axe rule fo rCas

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12887]
## Description
This task is for enabling the `scrollable-region-focusable` rule: Elements that have scrollable content should be accessible by keyboard

## Background/context/resources
See Epic #12884
